Assets/_BForBoss/_VFX/Scripts/TransformMover.cs

```diff
@@ -0,0 +1,55 @@
+//-----------------------------------------------------------------------
+// <copyright file="TransformMover.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Collections;
+using System.Collections.Generic;
+using PerigonGames;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace BForBoss
+{
+    public class TransformMover : MonoBehaviour
```

Comment:
AFAIK, this is for testing a dummy? i didn't open the project yet, but this is what i gotten from reading through the code?

Reply:
That's correct! This code is just for moving a dummy light to see how a normal map interacts with it. I just made it for my own testing purposes really.

Reply:
If you'll be specifically using for Light, then you can call this `LightSourceMover`
Or if you will apply it to different objects, (so it's more generic), u can call it `DemoPatrolBehaviour`

So it's specifically tells other developers it's for demo purposes, but it also tells them it's to only patrol between 2 points.